src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="content">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+      <p class="card__product-name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+
+      <p class="card__product-code">
+        Код товара: 195434
+      </p>
+
+      <div class="card__rating-container">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__feedbacks">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-word">
+          Цена:
+        </div>
+        <div class="card__price-number">
+          69 999 грн
+        </div>
```

Comment:
12 point from [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 

text can’t be inside div, you have so many tegs for text elements (h, p, span ...)

Reply:
fixed